<ui version="4.0" >
 <class>frmCreateUser</class>
 <widget class="QDialog" name="frmCreateUser" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>350</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>QNapi - zakładanie konta na serwerze NAPI-PROJEKT</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../res/resources.qrc" >
    <normaloff>:/icon/qnapi-48-icon.png</normaloff>:/icon/qnapi-48-icon.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="lbLogin" >
     <property name="text" >
      <string>Login:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="leLogin" />
   </item>
   <item row="0" column="2" >
    <widget class="QPushButton" name="pbCreate" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Załóż konto</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="lbPass" >
     <property name="text" >
      <string>Hasło:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="lePass" >
     <property name="echoMode" >
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="pbCancel" >
     <property name="text" >
      <string>Anuluj</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="lbRepeatPass" >
     <property name="text" >
      <string>Powtórz hasło:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="leRepeatPass" >
     <property name="echoMode" >
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="lbMail" >
     <property name="text" >
      <string>Adres e-mail:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QLineEdit" name="leMail" />
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="QLabel" name="lbStatus" >
     <property name="text" >
      <string>Wpisz dane potrzebne do założenia konta</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leLogin</tabstop>
  <tabstop>lePass</tabstop>
  <tabstop>leRepeatPass</tabstop>
  <tabstop>leMail</tabstop>
  <tabstop>pbCreate</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../res/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>frmCreateUser</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>434</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel" >
     <x>248</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
